functions-and-scope.js

```diff
@@ -53,13 +84,29 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // averageGrade([6, 4, 5]) geeft xxxx
 // averageGrade([8, 9, 4, 6, 10]) geeft xxxx
 
+function averageGrade(grades) {
+
+    let average = 0
+
+    for (let grade of grades) {
+        average = average + grade
```

Comment:
Je kunt dit korter schrijven
```suggestion
        average += grade;
```